go.mod

```diff
@@ -6,18 +6,18 @@ require (
 	github.com/go-logr/logr v1.2.3
 	github.com/jpillora/backoff v1.0.0 // indirect
 	github.com/mitchellh/go-homedir v1.1.0 // indirect
-	github.com/onsi/gomega v1.22.1
+	github.com/onsi/gomega v1.24.1
 	github.com/pkg/errors v0.9.1
 	github.com/sirupsen/logrus v1.9.0 // indirect
 	github.com/stretchr/testify v1.8.0
-	k8s.io/api v0.25.4
-	k8s.io/apimachinery v0.25.4
-	k8s.io/client-go v0.25.4
-	sigs.k8s.io/controller-runtime v0.13.1
+	k8s.io/api v0.26.1
+	k8s.io/apimachinery v0.26.2
+	k8s.io/client-go v0.26.1
+	sigs.k8s.io/controller-runtime v0.14.4
 )
 
 require (
-	github.com/serverlessworkflow/sdk-go/v2 v2.1.3-0.20221010134200-b9db1ddbffb2
+	github.com/serverlessworkflow/sdk-go/v2 v2.2.3-0.20230504180823-1e70f47d5804
```

Comment:
I think we need to release a new tag? Or did u?

Reply:
I didn't create the tag, should I, or will you?

Reply:
You can do it :)

Reply:
Created, please validate: https://github.com/serverlessworkflow/sdk-go/releases/tag/v.2.2.3